tests/testthat/test-cache.R

```diff
@@ -86,9 +86,16 @@ test_that("load_cache returns an empty environment if reading cache file fails",
   cache_f1 <- file.path(d1, fhash(f1))
   writeLines(character(), cache_f1)
 
-  expect_warning(e2 <- load_cache(file = f1, path = d1))
+  expect_warning(e2 <- load_cache(file = f1, path = d1), "Could not load cache file")
   saveRDS(e1, cache_f1)
-  expect_warning(e3 <- load_cache(file = f1, path = d1))
+  expect_warning(
+    expect_warning(
```

Comment:
Alternatively, we could `suppressWarnings()` the two R warnings unrelated to the test. LMK what you prefer.